packages/vuetify/src/mixins/elevatable.ts

```diff
@@ -12,7 +12,7 @@ export default Vue.extend({
       return this.elevation
     },
     elevationClasses (): Record<string, boolean> {
-      if (!this.computedElevation) return {}
+      if (!this.computedElevation && this.computedElevation !== 0) return {}
```

Comment:
Why not simply duplicate what we did for v2?

Reply:
Sure. I'll fix it

Reply:
done